ui/js/models/resultsets_store.js

```diff
@@ -67,7 +67,10 @@ treeherder.factory('ThResultSetStore', [
 
             // these params will be passed in each time we poll to remain
             // within the constraints of the URL params
-            var rsPollingParams = _.pick($location.search(), rsPollingKeys);
+            const locationSearch = $location.search();
```

Comment:
This has the same issue as above.  You could either add a check in the ``reduce`` or perhaps a ``filter`` afterward.  These objects are small enough that either way will work.